extension/DataX/doriswriter/src/main/java/com/alibaba/datax/plugin/writer/doriswriter/DorisJsonCodec.java

```diff
@@ -46,5 +44,4 @@ public String serialize(final Record row) {
         }
         return JSON.toJSONString(rowMap);
     }
-
-}
\ No newline at end of file
+}
```

Comment:
We can create `final Map<String, Object> rowMap` in constructor of `DorisJsonCodec` and reuse it in `serialize()`.
No need to create it for every row.